### PR TITLE
AWS Organizations

### PR DESCRIPTION
**Pull request by jlevypaloalto**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-4270)

## Description
Create the AWS Organizations integration.

Commands:
- [x] aws-org-root-list
- [x] aws-org-children-list
- [x] aws-org-parent-list
- [x] aws-org-organization-unit-get
- [x] aws-org-account-list
- [x] aws-org-organization-get
- [x] aws-org-account-remove
- [x] aws-org-account-move
- [x] aws-org-account-create
- [x] aws-org-account-close
- [x] aws-org-organization-unit-create
- [x] aws-org-organization-unit-delete
- [x] aws-org-organization-unit-rename
- [x] aws-org-policy-list
- [x] aws-org-target-policy-list
- [x] aws-org-policy-get
- [x] aws-org-policy-delete
- [x] aws-org-policy-attach
- [x] aws-org-policy-target-list
- [x] aws-org-resource-tag-add
- [x] aws-org-resource-tag-list

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.yml [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+      type: String
+    - contextPath: AWS.Organizations.Parent.Type
+      description: The type of the parent entity.
+      type: String
+    - contextPath: AWS.Organizations.Parent.ChildId
+      description: The unique identifier (ID) of the OU or account of the child of the parent entity.
+      type: String
+  - name: aws-org-organization-unit-get
+    arguments:
+    - description: The unique identifier (ID) of the organizational unit that to retrieve details about.
```
Comment (JudahSchwartz) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.yml:153]:
```suggestion
    - description: The unique identifier (ID) of the organizational unit to retrieve details for.
```

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.py [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+''' CONSTANTS '''
+
+SERVICE_NAME = 'organizations'
+REGION = 'us-east-1'
+MAX_PAGINATION = 20
+
+''' HELPER FUNCTIONS '''
+
+
+def create_client_session(args: dict, params: dict) -> 'OrganizationsClient':
```
Comment (JudahSchwartz) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.py:21]: why is OrganizationsClient in quotes everywhere?

Comment (JudahSchwartz) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.py:21]:
```suggestion
def create_client(args: dict, params: dict) -> 'OrganizationsClient':
```

Comment (JudahSchwartz) [reply Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.py:21]: lets add in docstring that this starts the session.

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.py [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+        service=SERVICE_NAME,
+        region=args.get('region'),
+        role_arn=args.get('roleArn'),
+        role_session_name=args.get('roleSessionName'),
+        role_session_duration=args.get('roleSessionDuration'),
+    )
+
+
+def paginate(
+    paginator: 'Paginator', key_to_pages: str, limit=None, page_size=None, next_token=None, **kwargs
```
Comment (JudahSchwartz) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.py:61]: why is paginator in quotes

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.py [hunk 4]
```diff
@@ -0,0 +1,322 @@
+from CommonServerPython import *  # noqa: E402
+import demistomock as demisto  # noqa: E402
+from AWSApiModule import *
+from typing import TYPE_CHECKING
+
+# The following imports are used only for type hints and autocomplete.
+# They are not used at runtime, and are not in the docker image.
+if TYPE_CHECKING:
+    from mypy_boto3_organizations import *
+    from botocore.paginate import Paginator
```
Comment (JudahSchwartz) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.py:10]: why are we using imports that arent available in the image? Better practice just not to use these at all

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.py [hunk 5]
```diff
@@ -0,0 +1,322 @@
...
+        pages.extend(response.get(key_to_pages, []))
+        next_token = response.get('NextToken')
+        if len(pages) >= max_items:
+            break
+
+    del pages[max_items:]
+    return pages, next_token
+
+
+def next_token_output_dict(outputs_prefix: str, next_token: str | None, page_outputs: Any, page_outputs_key: str = 'Id') -> dict:
```
Comment (JudahSchwartz) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.py:93]: can we use the code you added to CSP? Is that in yet?

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.py [hunk 6]
```diff
@@ -0,0 +1,322 @@
...
+    children, next_token = paginate(
+        aws_client.get_paginator('list_children'),
+        'Children',
+        limit=args.get('limit'),
+        next_token=args.get('next_token'),
+        page_size=args.get('page_size'),
+        ParentId=args['parent_id'],
+        ChildType={  # type: ignore
+            'Account': 'ACCOUNT',
+            'OrganizationalUnit': 'ORGANIZATIONAL_UNIT',
```
Comment (JudahSchwartz) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.py:144]: lets make ACCOUNT and ORGANIZATIONAL_UNIT constants

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.py [hunk 7]
```diff
@@ -0,0 +1,322 @@
...
+            'AWS Organization Unit',
+            ou.get('OrganizationalUnit', {}),
+            removeNull=True,
+        )
+    )
+
+
+def account_list_command(args: dict, aws_client: 'OrganizationsClient') -> CommandResults:
+
+    def JoinedTimestamp_to_str(account) -> dict:
```
Comment (JudahSchwartz) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.py:208]: move out of this function

**Review by JudahSchwartz — COMMENTED** [review]
Nice clean code! A couple comments, lets set a demo for final review.
Please move to docreview

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.py (hunk 7 above)
Comment (jlevypaloalto) [reply Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.py:208]: done

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.yml [hunk 8]
```diff
@@ -0,0 +1,262 @@
...
+      type: String
+    - contextPath: AWS.Organizations.Root.PolicyTypes.Status
+      description: The status of the policy type as it relates to the associated root.
+      type: String
+    - contextPath: AWS.Organizations.RootNextToken
+      description: If not null, indicates that more output is available than is included in the current response. Use this value in the next_token argument in a subsequent call of the command to get the next part of the output.
+      type: String
+  - name: aws-org-children-list
+    arguments:
+    - description: The unique identifier (ID) for the parent root or OU whose children are to be listed.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.yml:88]:
```suggestion
    - description: The unique identifier (ID) for the parent root or organizational unit (OU) whose children are to be listed.
```

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.yml [hunk 9]
```diff
@@ -0,0 +1,262 @@
...
+    description: List all of the organizational units (OUs) or accounts that are contained in the specified parent OU or root.
+    outputs:
+    - contextPath: AWS.Organizations.Children.Id
+      description: The unique identifier (ID) of the child entity.
+      type: String
+    - contextPath: AWS.Organizations.Children.Type
+      description: The type of the child entity.
+      type: String
+    - contextPath: AWS.Organizations.Children.ParentId
+      description: The unique identifier (ID) for the parent root or OU of the child entity.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.yml:120]:
```suggestion
      description: The unique identifier (ID) for the parent root or organizational unit of the child entity.
```

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.yml [hunk 10]
```diff
@@ -0,0 +1,262 @@
...
+      type: String
+    - contextPath: AWS.Organizations.ChildrenNextToken
+      description: If not null, indicates that more output is available than is included in the current response. Use this value in the next_token argument in a subsequent call of the command to get the next part of the output.
+      type: String
+  - name: aws-org-parent-list
+    arguments:
+    - description: |
+        The unique identifier (ID) of the OU or account whose parent containers you want to list. Don't specify a root.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.yml:128]:
```suggestion
        The unique identifier (ID) of the organizational unit or account whose parent containers you want to list. Don't specify a root.
```

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.yml [hunk 11]
```diff
@@ -0,0 +1,262 @@
...
+      This command returns only the immediate parents in the hierarchy.
+    outputs:
+    - contextPath: AWS.Organizations.Parent.Id
+      description: The unique identifier (ID) of the parent entity.
+      type: String
+    - contextPath: AWS.Organizations.Parent.Type
+      description: The type of the parent entity.
+      type: String
+    - contextPath: AWS.Organizations.Parent.ChildId
+      description: The unique identifier (ID) of the OU or account of the child of the parent entity.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.yml:149]:
```suggestion
      description: The unique identifier (ID) of the organizational unit or account of the child of the parent entity.
```

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.yml [hunk 12]
```diff
@@ -0,0 +1,262 @@
...
+      type: String
+    - contextPath: AWS.Organizations.OrganizationalUnit.Arn
+      description: The Amazon Resource Name (ARN) of the organizational unit.
+      type: String
+    - contextPath: AWS.Organizations.OrganizationalUnit.Name
+      description: The friendly name of the organizational unit.
+      type: String
+  - name: aws-org-account-list
+    arguments:
+    - description: Get a specific account by Id.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.yml:177]:
```suggestion
    - description: Get a specific account by Id.
```
```suggestion
    - description: Get a specific account by ID.
```

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.yml [hunk 13]
```diff
@@ -0,0 +1,262 @@
...
+      name: page_size
+    - description: The token denoting the next page of accounts, as given by the response of the previous run of this command under the context key "AWS.Organizations.AccountNextToken".
+      name: next_token
+    - description: The Amazon Resource Name (ARN) of the role to assume.
+      name: roleArn
+    - description: An identifier for the assumed role session.
+      name: roleSessionName
+    - description: The duration, in seconds, of the role session. The value can range from 900 seconds (15 minutes) up to the maximum session duration setting for the role.
+      name: roleSessionDuration
+    description: Lists all the accounts in the organization or a specific account by Id.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.yml:192]:
```suggestion
    description: Lists all the accounts in the organization or a specific account by ID.
```

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md [hunk 14]
```diff
@@ -0,0 +1,423 @@
...
+
+#### Base Command
+
+`aws-org-children-list`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| parent_id | The unique identifier (ID) for the parent root or OU whose children are to be listed. | Required |
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md:108]:
```suggestion
| parent_id | The unique identifier (ID) for the parent root or organizational unit whose children are to be listed. | Required | 
```

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md [hunk 15]
```diff
@@ -0,0 +1,423 @@
...
+| roleSessionName | An identifier for the assumed role session. | Optional | 
+| roleSessionDuration | The duration, in seconds, of the role session. The value can range from 900 seconds (15 minutes) up to the maximum session duration setting for the role. | Optional | 
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| AWS.Organizations.Children.Id | String | The unique identifier \(ID\) of the child entity. | 
+| AWS.Organizations.Children.Type | String | The type of the child entity. | 
+| AWS.Organizations.Children.ParentId | String | The unique identifier \(ID\) for the parent root or OU of the child entity. |
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md:123]:
```suggestion
| AWS.Organizations.Children.ParentId | String | The unique identifier \(ID\) for the parent root or organizational unit of the child entity. | 
```

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md [hunk 16]
```diff
@@ -0,0 +1,423 @@
...
+
+#### Base Command
+
+`aws-org-parent-list`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| child_id | The unique identifier (ID) of the OU or account whose parent containers you want to list. Don't specify a root.<br/>This value can be retrieved by running the command "aws-org-account-list". | Required |
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md:167]:
```suggestion
| child_id | The unique identifier (ID) of the organizational unit or account whose parent containers you want to list. Don't specify a root.<br/>This value can be retrieved by running the command "aws-org-account-list". | Required | 
```

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md [hunk 17]
```diff
@@ -0,0 +1,423 @@
...
+| roleSessionName | An identifier for the assumed role session. | Optional | 
+| roleSessionDuration | The duration, in seconds, of the role session. The value can range from 900 seconds (15 minutes) up to the maximum session duration setting for the role. | Optional | 
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| AWS.Organizations.Parent.Id | String | The unique identifier \(ID\) of the parent entity. | 
+| AWS.Organizations.Parent.Type | String | The type of the parent entity. | 
+| AWS.Organizations.Parent.ChildId | String | The unique identifier \(ID\) of the OU or account of the child of the parent entity. |
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md:178]:
```suggestion
| AWS.Organizations.Parent.ChildId | String | The unique identifier \(ID\) of the organizational unit or account of the child of the parent entity. | 
```

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md [hunk 18]
```diff
@@ -0,0 +1,423 @@
...
+>### AWS Organization Unit
+>|Arn|Id|Name|
+>|---|---|---|
+>| arn:aws:organizations::111222333444:ou/o-abcde12345/ou-ab12-abcd1234 | ou-ab12-abcd1234 | Moishy OU |
+
+
+### aws-org-account-list
+
+***
+Lists all the accounts in the organization or a specific account by Id.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md:261]:
```suggestion
Lists all the accounts in the organization or a specific account by ID.
```

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md [hunk 19]
```diff
@@ -0,0 +1,423 @@
...
+
+#### Base Command
+
+`aws-org-account-list`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| account_id | Get a specific account by Id. | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md:271]:
```suggestion
| account_id | Get a specific account by ID. | Optional | 
```

File: Packs/AWS-Organizations/README.md [hunk 20]
```diff
@@ -0,0 +1,11 @@
...
+This content pack allows you to manage AWS Organizations from Cortex XSOAR.
+
+The AWS Organizations pack enables you to:
+
+- List AWS accounts in an organization
+- Get details about specific accounts
+
+#### Use Cases
+
+- **Account auditing** - Regularly list all accounts in the organization and fetch details like IAM users, roles, etc.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Organizations/README.md:11]:
```suggestion
**Account auditing** - Regularly list all accounts in the organization and fetch details like IAM users, roles, etc.
```

**Comment by ShirleyDenkberg** [comment]
@JudahSchwartz @michal-dagan Doc review completed.

File: Packs/AWS-Organizations/README.md [hunk 21]
```diff
@@ -0,0 +1,28 @@
...
+
+The AWS Organizations pack enables you to:
+
+- List AWS accounts in an organization
+- Get details about specific accounts
+- Create new AWS accounts
+- Move accounts between organizational units
+- Manage policies for accounts
+- Invite new accounts to join the organization
+- Tag resources across the organization
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Organizations/README.md:12]:
```suggestion
- List AWS accounts in an organization.
- Get details about specific accounts.
- Create new AWS accounts.
- Move accounts between organizational units.
- Manage policies for accounts.
- Invite new accounts to join the organization.
- Tag resources across the organization.
```

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md [hunk 22]
```diff
@@ -0,0 +1,1119 @@
...
+| email | The email address of the owner to assign to the new member account. This email address must not already be associated with another Amazon Web Services account. Use a valid email address to complete account creation. | Required | 
+| iam_user_access_to_billing | If set to ALLOW, the new account enables IAM users to access account billing information if they have the required permissions. If set to DENY, only the root user of the new account can access account billing information. Possible values are: Allow, Deny. Default is Allow. | Optional | 
+| role_name | The name of an IAM role that AWS Organizations automatically pre-configures in the new member account. This role trusts the management account, allowing users in the management account to assume the role, as permitted by the management account administrator. The role has administrator permissions in the new member account. Default is OrganizationAccountAccessRole. | Optional | 
+| tags | A comma-separated list of tags to attach to the newly created account. Each tag should be in the format: "key=value". | Required | 
+| request_id | The Id of the create request that is used for polling. | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md:983]:
```suggestion
| request_id | The ID of the create request that is used for polling. | Optional | 
```

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md [hunk 23]
```diff
@@ -0,0 +1,1119 @@
...
+| tags | A comma-separated list of tags to attach to the newly created account. Each tag should be in the format: "key=value". | Required | 
+| request_id | The Id of the create request that is used for polling. | Optional | 
+| interval_in_seconds | Indicates how long to wait between command executions (in seconds) when the 'polling' argument is true. Minimum value is 10 seconds. Default is 30. | Optional | 
+| timeout | Indicates the time in seconds until the polling sequence times out. Default is 600. | Optional | 
+| roleArn | The Amazon Resource Name (ARN) of the role to assume. | Optional | 
+| roleSessionName | An identifier for the assumed role session. | Optional | 
+| roleSessionDuration | The duration, in seconds, of the role session. The value can range from 900 seconds (15 minutes) up to the maximum session duration setting for the role. | Optional | 
+
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md:989]: In the YML file there is an argument called hide_polling_output (line 333).  This argument does not appear here in the README.

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md [hunk 24]
```diff
@@ -0,0 +1,1119 @@
...
+`aws-org-organization-unit-create`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| name | The friendly name to assign to the new OU. | Required | 
+| parent_id | The unique identifier (ID) of the parent root or OU to create the new OU in. | Required | 
+| tags | A comma-separated list of tags to attach to the newly created OU. Each tag should be in the format: "key=value". | Required |
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md:439]:
```suggestion
| name | The friendly name to assign to the new organizational unit. | Required | 
| parent_id | The unique identifier (ID) of the parent root or organizational unit to create the new organizational unit in. | Required | 
| tags | A comma-separated list of tags to attach to the newly created organizational unit. Each tag should be in the format: "key=value". | Required | 
```

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md [hunk 25]
```diff
@@ -0,0 +1,1119 @@
...
+| roleSessionName | An identifier for the assumed role session. | Optional | 
+| roleSessionDuration | The duration, in seconds, of the role session. The value can range from 900 seconds (15 minutes) up to the maximum session duration setting for the role. | Optional | 
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| AWS.Organizations.OrganizationUnit.Id | String | The unique identifier \(ID\) associated with this OU. | 
+| AWS.Organizations.OrganizationUnit.Arn | String | The Amazon Resource Name \(ARN\) of this OU. | 
+| AWS.Organizations.OrganizationUnit.Name | String | The friendly name of this OU. |
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md:450]:
```suggestion
| AWS.Organizations.OrganizationUnit.Id | String | The unique identifier \(ID\) associated with this organizational unit. | 
| AWS.Organizations.OrganizationUnit.Arn | String | The Amazon Resource Name \(ARN\) of this organizational unit. | 
| AWS.Organizations.OrganizationUnit.Name | String | The friendly name of this organizational unit. | 
```

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md [hunk 26]
```diff
@@ -0,0 +1,1119 @@
...
+#### Base Command
+
+`aws-org-organization-unit-rename`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| organizational_unit_id | The unique identifier (ID) of the OU to rename. This value can be retrieved by running the command "aws-org-parent-list". | Required | 
+| name | The new name to assign to the OU. | Required |
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md:490]:
```suggestion
| name | The new name to assign to the organizational unit. | Required | 
```

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md [hunk 27]
```diff
@@ -0,0 +1,1119 @@
...
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| AWS.Organizations.Policy.Id | String | The unique identifier \(ID\) of the policy. | 
+| AWS.Organizations.Policy.Arn | String | The Amazon Resource Name \(ARN\) of the policy. | 
+| AWS.Organizations.Policy.Name | String | The friendly name of the policy. | 
+| AWS.Organizations.Policy.Description | String | The description of the policy. | 
+| AWS.Organizations.Policy.Type | String | The type of policy. | 
+| AWS.Organizations.Policy.AwsManaged | Boolean | Indicates whether the specified policy is an Amazon Web Services managed policy. If true, the policy can be attached to roots, OUs, or accounts, but cannot be edited. |
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md:569]:
```suggestion
| AWS.Organizations.Policy.AwsManaged | Boolean | Indicates whether the specified policy is an Amazon Web Services managed policy. If true, the policy can be attached to roots, organizational units, or accounts, but cannot be edited. | 
```

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md [hunk 28]
```diff
@@ -0,0 +1,1119 @@
...
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md:638]:
```suggestion
| AWS.Organizations.Policy.AwsManaged | Boolean | Indicates whether the specified policy is an Amazon Web Services managed policy. If true, the policy can be attached to roots, organizational units, or accounts, but cannot be edited. | 
```

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md [hunk 29]
```diff
@@ -0,0 +1,1119 @@
...
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| AWS.Organizations.TargetPolicy.Id | String | The unique identifier \(ID\) of the policy. | 
+| AWS.Organizations.TargetPolicy.Arn | String | The Amazon Resource Name \(ARN\) of the policy. | 
+| AWS.Organizations.TargetPolicy.Name | String | The friendly name of the policy. | 
+| AWS.Organizations.TargetPolicy.Description | String | The description of the policy. | 
+| AWS.Organizations.TargetPolicy.Type | String | The type of policy. | 
+| AWS.Organizations.TargetPolicy.AwsManaged | Boolean | Indicates whether the specified policy is an Amazon Web Services managed policy. If true, the policy can be attached to roots, OUs, or accounts, but cannot be edited. |
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md:789]:
```suggestion
| AWS.Organizations.TargetPolicy.AwsManaged | Boolean | Indicates whether the specified policy is an Amazon Web Services managed policy. If true, the policy can be attached to roots, organizational units, or accounts, but cannot be edited. | 
```

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md [hunk 30]
```diff
@@ -0,0 +1,1119 @@
...
+>### AWS Organization *ou-ab12-abcd1234* Policies
+>|Id|Arn|Name|Description|Type|AwsManaged|
+>|---|---|---|---|---|---|
+>| p-1234abcd | arn:aws:organizations::111222333444:policy/o-abcde12345/service_control_policy/p-1234abcd | Test | Used for test purposes | SERVICE_CONTROL_POLICY | false |
+>| p-FullAWSAccess | arn:aws:organizations::aws:policy/service_control_policy/p-FullAWSAccess | FullAWSAccess | Allows access to every operation | SERVICE_CONTROL_POLICY | true |
+
+### aws-org-policy-delete
+
+***
+Deletes the specified policy from the organization. Before performing this operation, the policy must be detached from all organizational units (OUs), roots, and accounts.'
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md:837]:
```suggestion
Deletes the specified policy from the organization. Before performing this operation, the policy must be detached from all organizational units (OUs), roots, and accounts.
```

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md [hunk 31]
```diff
@@ -0,0 +1,1119 @@
...
+
+#### Base Command
+
+`aws-org-resource-tag-add`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| resource_id | The ID of the resource to add a tag to. This value can be retrieved by running the commands "aws-org-root-list", "aws-org-account-list", "aws-org-root-list" or "aws-org-policy-list". | Required |
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md:878]:
```suggestion
| resource_id | The ID of the resource to add a tag to. This value can be retrieved by running the command "aws-org-root-list", "aws-org-account-list", "aws-org-root-list", or "aws-org-policy-list". | Required | 
```

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md [hunk 32]
```diff
@@ -0,0 +1,1119 @@
...
+
+#### Base Command
+
+`aws-org-resource-tag-list`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| resource_id | The ID of the resource with the tags to list. This value can be retrieved by running the commands "aws-org-root-list", "aws-org-account-list", "aws-org-root-list" or "aws-org-policy-list". | Required |
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md:911]:
```suggestion
| resource_id | The ID of the resource with the tags to list. This value can be retrieved by running the command "aws-org-root-list", "aws-org-account-list", "aws-org-root-list", or "aws-org-policy-list". | Required | 
```

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md [hunk 33]
```diff
@@ -0,0 +1,1119 @@
...
+#### Base Command
+
+`aws-org-policy-attach`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| policy_id | The unique identifier (ID) of the policy to attach to the target. This value can be retrieved by running the command "aws-org-policy-list". | Required | 
+| target_id | The unique identifier (ID) of the root, OU, or account to attach the policy to. This value can be retrieved by running the commands "aws-org-root-list" or "aws-org-account-list". | Required |
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md:681]:
```suggestion
| target_id | The unique identifier (ID) of the root, organizational unit, or account to attach the policy to. This value can be retrieved by running the command "aws-org-root-list" or "aws-org-account-list". | Required | 
```

**Comment by ShirleyDenkberg** [comment]
@JudahSchwartz As requested, I reviewed the files again.

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md (hunk 23 above)
Comment (jlevypaloalto) [reply Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md:989]: it's  a hidden argument

File: Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.yml [hunk 34]
```diff
@@ -0,0 +1,699 @@
...
+      name: timeout
+    - description: The Amazon Resource Name (ARN) of the role to assume.
+      name: roleArn
+    - description: An identifier for the assumed role session.
+      name: roleSessionName
+    - description: The duration, in seconds, of the role session. The value can range from 900 seconds (15 minutes) up to the maximum session duration setting for the role.
+      name: roleSessionDuration
+    - name: hide_polling_output
+      hidden: true
+      description: ''
```
[the remark this answers is not in the document]
Comment (jlevypaloalto) [reply Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.yml:334]: it's a hidden arg

**Review by JudahSchwartz — APPROVED** [review]
🚀 great work!

**Pull request merged**